sea-orm-macros/src/derives/column.rs

```diff
@@ -41,6 +41,44 @@ pub fn impl_default_as_str(ident: &Ident, data: &Data) -> syn::Result<TokenStrea
     ))
 }
 
+pub fn impl_col_from_str(ident: &Ident, data: &Data) -> syn::Result<TokenStream> {
+    let parse_error_iden = format_ident!("Parse{}Err", ident);
+
+    let data_enum = match data {
+        Data::Enum(data_enum) => data_enum,
+        _ => {
+            return Ok(quote_spanned! {
+                ident.span() => compile_error!("you can only derive DeriveColumn on enums");
+            })
+        }
+    };
+
+    let columns = data_enum.variants.iter().map(|column| {
+        let column_iden = column.ident.clone();
+        let column_str_snake = column_iden.to_string().to_snake_case();
+        let column_str_mixed = column_iden.to_string().to_mixed_case();
```

Comment:
Why do you want to favour MixedCase over CamelCase?
The different between them is in Camel, the first character is also captialized:
```
CamelCase
mixedCase
```

Reply:
Yes because in my case graphql uses `mixedCase`.

I actually always thought camel case was `camelCase`, not `CamelCase`.

Reply:
Good to know. So the GraphQL convention is to use `camelCase`. Well maybe that's a `heck` naming thing.